### PR TITLE
Allow testing nda renewal notifications.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/groups/tasks.py [hunk 1]
```diff
@@ -161,16 +162,17 @@ def notify_membership_renewal():
                        .exclude(userprofile__id__in=curator_ids))
 
         # Filter memberships to be notified
-        last_update_days = group.invalidation_days - days_prior_invalidation
-        last_update = now() - timedelta(days=last_update_days)
+        if switch_is_active('test_group_invalidation_email'):
```
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: Shouldn't this be `if not`?

File: mozillians/groups/views.py [hunk 2]
```diff
@@ -608,3 +609,10 @@ def force_group_invalidation(request, url, alias_model, template=''):
         raise Http404
 
     return redirect(reverse('groups:show_group', args=[group.url]))
+
+
+@never_cache
+@waffle_switch('test_membership_renewal_notification')
+def membership_renewal_notification(request):
```
Comment (akatsoulas) [inline mozillians/groups/views.py]: Let's change the view to `test_membership_renewal_notification` or add a docstring explaining why is this view here and what it does.

File: mozillians/groups/tasks.py (hunk 1 above)
Comment (akatsoulas) [inline mozillians/groups/tasks.py]: Let's add a comment to this if clause.

**Pull request merged**